src/renderer/entities/operations/ui/SignButton.tsx

```diff
@@ -42,7 +45,7 @@ export const SignButton = ({ disabled, type, className, onClick }: Props) => {
       }
       onClick={onClick}
     >
-      {t(type ? WalletText[type as SignableWalletFamily] : UnkownWalletText)}
+      {t(type && !isDefault ? WalletText[type as SignableWalletFamily] : DefaultSignButton)}
```

Comment:
As I understand if we have 2 buttons, `SignButton` must have simplified naming - "Sign", right?

Reply:
Right